### ISSUE TITLE
Fritz Chess 17 Steam Edition (1279590)

### ISSUE THREAD

**Issue by ghost** [ValveSoftware/Proton#4596]
# Compatibility Report
- Name of the game with compatibility issues: Fritz17

## System Information
- GPU: Radeon Vega Mobile Gfx (AMD AMD Radeon(TM) Vega 10 Graphics)
- Driver/LLVM version: Mesa 20.3.3
- Kernel version: 5.10.12
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/ardmarel/e4aa6e8195a3324f08155b921ca5a9f4
- Proton version: 5.13.5 and 5.0.10

## I confirm:
- [X] that I haven't found an existing compatibility report for this game.
- [X] that I have checked whether there are updates for my system available.

## Symptoms 
The application fails to launch correctly

## Reproduction
Launching the application

**Comment by ghost** [comment]
[steam-1279590.log](https://github.com/ValveSoftware/Proton/files/5936934/steam-1279590.log)

**Comment by kisak-valve** [comment]
Hello @ardmarel, `wine: Call from 000000007B01193E to unimplemented function api-ms-win-crt-multibyte-l1-1-0.dll._mbslwr_s_l, aborting` from the log looks like a line of interest.

**Comment by ghost** [comment]
O wow, i did miss that :joy: thanks for the notice. A quick search show me it has something to do with the Visual C++ redistributable for Visual Studio 2015.  i'm going to try some protontricks things to hopefully get it working

**Comment by ghost** [comment]
A reinstall and some attemps later a first time launch opens what installs the Visual C++ 2015-2019 redistributable for Visual (32 and 64 bit) giving me the following log
[steam-1279590.log](https://github.com/ValveSoftware/Proton/files/5937100/steam-1279590.log)

giving me the following
`3874.590:00cc:00d0:err:module:import_dll Library OpenImageDenoise.dll (which is needed by L"Z:\\home\\ard\\.local\\share\\Steam\\steamapps\\common\\FritzChess 17 SteamEdition\\ChessProgram17\\ChessProgram17.exe") not found

3874.592:00cc:00d0:err:module:import_dll Library ospray.dll (which is needed by L"Z:\\home\\ard\\.local\\share\\Steam\\steamapps\\common\\FritzChess 17 SteamEdition\\ChessProgram17\\ChessProgram17.exe") not found
`

at this point i'm not sure how to continue. just searching further.

**Comment by kisak-valve** [comment]
The previous log has `trace:loaddll:build_module Loaded L"Z:\\home\\ard\\.local\\share\\Steam\\steamapps\\common\\FritzChess 17 SteamEdition\\ChessProgram17\\Denoise\\OpenImageDenoise.dll" at 0000000180000000: native`
`trace:loaddll:build_module Loaded L"Z:\\home\\ard\\.local\\share\\Steam\\steamapps\\common\\FritzChess 17 SteamEdition\\ChessProgram17\\Raytracer\\ospray.dll" at 0000000000A60000: native` which is missing in the new log, so I suspect that's an issue with your workaround.

**Comment by ghost** [comment]
Yeah, its weired. now it just gives the error it cannot be loaded while the files are there. Another reinstall and i'm still unable to reproduce the situation there the dll's did load.
